pkgs/servers/nominatim/default.nix

```
@@ -1,7 +1,9 @@
 { stdenv, lib, fetchFromGitHub, fetchurl
-, clang-tools, cmake, bzip2, zlib, expat, boost, git, pandoc, gzip
+, clang-tools, cmake, bzip2, zlib, expat, boost, git, pandoc
 , postgresql_12
-, python3, python3Packages, php
+# Nominatim needs to be built with the same postgres version it will target
+, postgres ? postgresql_12
```

```suggestion
, postgresql
```

Then add the override `postgresql = postgresql_12;` in `all-packages.nix`.

---

Or do you mean when do the callPackage of nominatim?

---

> There already is an attribute postgresql, but it's set to postgresql_14

Yes, that's why the additional override in `all-packages.nix` to set `postgresql_12` by default.